src/main/scala/com/mozilla/telemetry/pings/package.scala

```diff
@@ -161,6 +161,19 @@ package object pings {
       new Timestamp(this.Timestamp / 1000000)
     }
 
+    def normalizedBuildId(): Option[String] = {
+      `environment.build`.flatMap(_.buildId) match {
+        case Some(buildId: String) => {
+          val buildIdDay = buildId.slice(0, 6).toString()
```

Comment:
This is taking the `YYYYMM` of the build, you need `YYYYMMDD`

Reply:
Right you are